packages/hardhat-chai-matchers/src/internal/addressable.ts

```diff
@@ -1,4 +1,4 @@
-import { isAddress, isAddressable } from "ethers";
+import type EthersT from "ethers";
```

Comment:
This has a significant impact in performance, I think we should add a changeset for it, at least for the chai matchers (the change in core also improves the loading performance, but not that much).